SortApp/SortApp.BL.Test/Sorting/BaseSortTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SortApp.BL.Sortings;
+
+namespace SortApp.BL.Test.Sorting
+{
+	[TestClass]
+	public abstract class BaseSortTests
+	{
+		/// <summary>
+		/// Creates implementation with specific sorting algorithm.
+		/// </summary>
+		/// <owner>Oleh Petrenko</owner>
+		protected abstract Sorter<int> CreateSorter();
+
+		/// <summary>
+		/// Performs a base logic for test towards the logics of sorting algorithm when incoming array is empty.
+		/// </summary>
+		/// <owner>Oleh Petrenko</owner>
+		[TestMethod]
+		public void SortEmptyArray()
```

Comment:
Please, use suffix Test on names of test methods. in this case it would be SortEmptyArrayTest